src/layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Navbar, Container } from 'react-bootstrap';
+import { useLocation, useNavigate } from 'react-router-dom';
+import '../../assets/styles/dashboard.scss';
+
+const buttons = [
+    { icon: 'fas fa-home', text: 'Home', action: '/' },
+    { icon: 'far fa-address-card', text: 'Address', action: '/address' },
+    { icon: 'fas fa-piggy-bank', text: 'Page 2', action: '/bank' },
+    { icon: 'fas fa-heartbeat', text: 'Heartbeat', action: '/heartbeat' }
+];
+
+const Footer = () => {
+    const { pathname } = useLocation();
+    const navigate = useNavigate();
+
+    const getButtonClass = (action: string) => {
```

Comment:
you can do this job with a package called `clsx` its better for conditional classes